### PR TITLE
UIOR-383 Edit POL from view POL in the three pane POL search layout

### PR DESCRIPTION
**Pull request by Feboll**
## Purpose

https://issues.folio.org/browse/UIOR-383

## Approach

add transition to edit from POL details view

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/components/POLine/POLineView.js [hunk 1]
```diff
@@ -110,10 +111,14 @@ class POLineView extends Component {
 
   onEditPOLine = (e) => {
     if (e) e.preventDefault();
-    const { queryMutator } = this.props;
+    const { queryMutator, order, line } = this.props;
 
-    queryMutator.update({ layer: 'edit-po-line' });
-  }
+    queryMutator.update({
```
Comment (NikitaSedyx) [inline src/components/POLine/POLineView.js]: please try to avoid query mutator

File: src/components/POLine/POLineView.js [hunk 2]
```diff
@@ -110,10 +111,14 @@ class POLineView extends Component {
...
+      _path: `/orders/view/${order.id}/po-line/view/${line.id}`,
+      layer: 'edit-po-line',
+      filters: `${FILTERS.STATUS}.${WORKFLOW_STATUS.open},${FILTERS.STATUS}.${WORKFLOW_STATUS.pending}`,
```
Comment (aliaksei-chumakou) [inline src/components/POLine/POLineView.js]: What the reason you've put filters here?

Comment (Feboll) [reply src/components/POLine/POLineView.js]: Updated

File: src/components/POLine/POLineView.js (hunk 1 above)
Comment (Feboll) [reply src/components/POLine/POLineView.js]: I tried, but case with `history.push` don't work here.

**Comment by aliaksei-chumakou** [comment]
✖ 1 test failed

File: src/components/POLine/POLineView.js [hunk 3]
```diff
@@ -43,6 +43,7 @@ import {
   PHRESOURCES,
   OTHER,
 } from './const';
+import { FILTERS as ORDER_FILTERS, FILTERS } from '../../OrdersList';
```
Comment (aliaksei-chumakou) [inline src/components/POLine/POLineView.js]: is FILTERS used somewhere?

Comment (Feboll) [reply src/components/POLine/POLineView.js]: removed

**Pull request merged**